Report/sections/2_design/Scanner.tex

```diff
@@ -0,0 +1,26 @@
+\section{Scanner} \label{chap:scanner}
+
+The first phase in a compiler is the scanner, also called "lexical analysis" or "lexer".
+The purpose of the scanner is to interpret the meaning of each individual words in some sourcecode 
+by grouping them into different \emph{tokens}.
```

Comment:
A scanner doesn't interpret; the parser does. A scanner... "Begins the analysis of the source program by reading the input text (character by character) and grouping individual characters into tokens such as identifiers, integers, reserved words, and delimiters." (Fischer p. 45)